lib/brain.js

```diff
@@ -2252,7 +2257,7 @@ Brain.prototype._start = function _start () {
 
   this.browser().send({
     tx: this.tx,
-    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled'
+    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled',
```

Comment:
Any reason for this change? Only thing I could think of is the prettier config with `trailingComma: true`

Reply:
Or, thinking better about this change, seems as a leftover change where you added a field that was later deleted during development